arangod/Statistics/StatisticsFeature.cpp

```diff
@@ -144,6 +142,12 @@ DECLARE_HISTOGRAM(arangodb_client_connection_statistics_bytes_received,
                   BytesReceivedScale, "Bytes received for a request");
 DECLARE_HISTOGRAM(arangodb_client_connection_statistics_bytes_sent,
                   BytesSentScale, "Bytes sent for a request");
+DECLARE_HISTOGRAM(arangodb_client_user_connection_statistics_bytes_received,
+                  BytesReceivedScale,
+                  "Bytes received for a request, only user traffic");
```

Comment:
Nit. Although it would it inconsistent with other existing metrics, so please check if you want to apply the suggestion or not.
```suggestion
                  "Bytes received for requests, only user traffic");
```